app/models/manageiq/providers/embedded_ansible/provider.rb

```diff
@@ -6,7 +6,7 @@ class ManageIQ::Providers::EmbeddedAnsible::Provider < ::Provider
   has_one :automation_manager,
           :foreign_key => "provider_id",
           :class_name  => "ManageIQ::Providers::EmbeddedAnsible::AutomationManager",
-          :dependent   => :destroy,
+          :dependent   => :destroy, # to be removed after ansible_tower side code is updated
```

Comment:
Removal of this will break the spec. We'll need https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/54 (which is in circular dependency with this one).

If we know we'll merge both at the same(near) time, I can do this now.

Reply:
I assume you meant to link https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/49 but since that has its own `has_one :automation_manager` and it's a different subclass of `Provider`, I don't see how they're related.

Reply:
No, specifically needed is this fix of the factory object https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/49/files#diff-58d459079c55b950f3e93005074af740R25

Reply:
I'm very lost now..

Reply:
Offline explained this to Brandon that this has to do with an existing bug in the shared Tower spec (linked in my last comment)